docs/SEEEDSTUDIO_WIO_E5.md

```diff
@@ -4,6 +4,33 @@
 
 The SeeedStudio [Wio-E5 module](https://wiki.seeedstudio.com/LoRa-E5_STM32WLE5JC_Module) is a highly attractive module for building mLRS equipment. SeeedStudio provides a number of boards which are based on this module, and which are quite interesting hardware for mLRS. However, these boards are not perfect since their pins are not ready-made for the purposes of mLRS. So, some tweaking and (easy) soldering is required.
 
+## Flashing the Modules ##
+
+Important - every time that you power a board you should ensure that there is an antenna connected otherwise you risk damaging the RF section.
```

Comment:
maybe "Important: Every..:", 
maybe this shoul be highlighted, like bold and italics
just suggestions

Reply:
Now bold